modules/src/applications/transfer/relay/on_timeout_packet.rs

```diff
@@ -0,0 +1,13 @@
+use crate::applications::transfer::context::Ics20Context;
+use crate::applications::transfer::error::Error as Ics20Error;
+use crate::applications::transfer::packet::PacketData;
+use crate::applications::transfer::relay::refund_packet_token;
+use crate::core::ics04_channel::packet::Packet;
+
+pub fn process_timeout_packet(
```

Comment:
Similarly, can we inline this function where it is called in `on_timeout_packet()`?